### DISCUSSION TITLE
Obliquity of ecliptic

### DISCUSSION THREAD

**Discussion by Max55max** [Stellarium/stellarium#3606] [General]
Hi,
I noted that the values of the ecliptic obliquity and that of the declination of the Sun in the date of the summer solstice (λ =90°- in date) as for distant past dates diverge (after 5000BC.)
 I would like to know how Stellarium calculates the value of the obliquity of ecliptic. I don’t find anyting regarding that in “The User Guide, Appendix F”
Thank you
Massimo Calabresi

**Comment by gzotti** [comment]
Please read Appendix F.5 (ecliptic obliquity and earth orientation are pretty closely related). If planet positions before 4000BC matter for you, install DE441. If planet positions before 13.000BC matter to you, please point us to the data.
